src/components/dictionaryConcepts/components/helperFunction.js

```diff
@@ -59,6 +59,14 @@ export const MAP_TYPE = {
   questionAndAnswer: 'Q-AND-A',
 };
 
+export const CONCEPT_TYPE = {
+  question: 'Question',
+};
+
+export const CLASS = {
```

Comment:
If the above is concept type, shouldn't this be concept class?

Reply:
I have changed it.